src/platforms/android/common/gradle.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: Gradle
+sidebar_order: 100
+description: "Learn about using the Sentry Android Gradle Plugin."
+redirect_from:
+  - /platforms/android/proguard/#gradle-configuration
+---
+
+The [Sentry Android Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) comes as an addition to the main Android and Hybrid SDKs and offers
+seamless integration with the Gradle build system. It supports uploading ProGuard/R8 mappings, uploading Native debug symbols and sources as well as tracing
+auto-instrumentation through bytecode manipulation.
```

Comment:
I know this probably sounds more awkward, but this should be a noun phrase instead of a verb phrase. Some ugly English grammar :-p
```suggestion
The [Sentry Android Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) is an addition to the main Android and Hybrid SDKs and offers
seamless integration with the Gradle build system. It supports the upload of ProGuard/R8 mappings and of Native debug symbols and sources, as well as
auto-instrumentation tracing through bytecode manipulation.
```